dll/win32/shimgvw/shimgvw.c

```diff
@@ -216,19 +253,29 @@ static VOID pLoadImage(LPCWSTR szOpenFileName)
     if (!g_pImage)
     {
         DPRINT1("GdipLoadImageFromFile() failed\n");
+        Preview_UpdateTitle(pData, NULL);
         return;
     }
 
-    Anime_LoadInfo(&g_Anime);
+    Anime_LoadInfo(&pData->m_Anime);
 
     if (szOpenFileName && szOpenFileName[0])
         SHAddToRecentDocs(SHARD_PATHW, szOpenFileName);
 
     /* Reset zoom and redraw display */
-    ResetZoom();
+    Preview_ResetZoom(pData);
+
+    Preview_UpdateTitle(pData, szOpenFileName);
 }
 
-static VOID pSaveImageAs(HWND hwnd)
+static VOID
+Preview_pLoadImageFromNode(PPREVIEW_DATA pData, SHIMGVW_FILENODE *pNode)
+{
+    return Preview_pLoadImage(pData, (pNode ? pNode->FileName : NULL));
```

Comment:
`shimgvw.c(274): error C4098: 'Preview_pLoadImageFromNode': 'void' function returning a value`
```suggestion
    Preview_pLoadImage(pData, (pNode ? pNode->FileName : NULL));
```